animations/ClipClasses.py

```diff
@@ -0,0 +1,62 @@
+from moviepy.editor import *
+from animations.BaseClip import BaseClip
+
+
+class videoClip(BaseClip):
+
+    def __init__(self, clip, start_time, end_time, fps= 29.97):
+        BaseClip.__init__(self, VideoFileClip(clip).subclip(start_time, end_time).set_fps(fps))
+
+
+class photoClip(BaseClip):
+    def __init__(self, image, duration=5):
+        BaseClip.__init__(self, ImageClip(img=image, duration=duration))
+
+
+class finalVideo(BaseClip):
+
+    def __init__(self):
+        self.clip = None
```

Comment:
Why doesn't this one inherit from ```BaseClip```?

Reply:
It does, but I didn't want to initialize it the same way the others do. 

For this one, you create the object empty and add in clips that are done.